redis-sink/src/main/resources/application.yml

```diff
@@ -0,0 +1,12 @@
+server:
+  port: 8081
+spring:
+  cloud:
+    stream:
+      bindings:
+        input: testtock
```

Comment:
Agreed, but one would only ever use this value without overriding it when they'd just run something like "A | B", in a dilettante fashion, no?
As soon as they want to create their real stream(s), they'd need to provide another value.